api_tests/dry/rfc003_swap_reject.js

```diff
@@ -12,8 +12,8 @@ const beta_ledger_name = "Ethereum";
 const beta_ledger_network = "regtest";
 
 const alpha_asset_name = "Bitcoin";
-const alpha_asset_reasonable_quantity = "100000000";
-const alpha_asset_stingy_quantity = "100";
+const alpha_asset_reasonable_quantity = "400000002";
```

Comment:
Undo this. This is a side effect of me trying to make comit-nodes not need to be restarted between each dry test.